javascript/javascript3/week2/visual-promise/move-element.js

```diff
@@ -0,0 +1,65 @@
+// Creating new scope
+{
+    const redBox = document.querySelector('ul.marks li:nth-child(1)');
+    const blueBox = document.querySelector('ul.marks li:nth-child(2)');
+    const greenBox = document.querySelector('ul.marks li:nth-child(3)');
```

Comment:
It works with `nth-child` but it would maybe suggest using ids or classes. because if for some reason the order is changing then it could result in this solution not working as expected